build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -345,27 +340,40 @@ def from_device_info_and_name(device_info: DeviceInfo, name: str):
     model_source = model_source.strip("()")
     model_tags = model_tags.strip("[]").split(",")
     bench_mode = bench_mode.split(",")
-    runner = IREE_PRETTY_NAMES_TO_DRIVERS.get(runner)
-    return BenchmarkInfo(model_name, model_tags, model_source, bench_mode,
-                         runner, device_info)
+
+    runner = IREE_PRETTY_NAME_TO_DRIVER_NAME.get(runner)
+    if not runner:
+      raise ValueError(f"Unrecognized runner: {runner}")
+
+    return BenchmarkInfo(model_name=model_name,
+                         model_tags=model_tags,
+                         model_source=model_source,
+                         bench_mode=bench_mode,
+                         driver_info=IREE_DRIVERS_INFOS[runner],
+                         device_info=device_info)
```

Comment:
This error message is is always going to print "None".

```suggestion
    driver = IREE_PRETTY_NAME_TO_DRIVER_NAME.get(runner)
    if not driver:
      raise ValueError(f"Unrecognized runner: {runner}")

    return BenchmarkInfo(model_name=model_name,
                         model_tags=model_tags,
                         model_source=model_source,
                         bench_mode=bench_mode,
                         driver_info=IREE_DRIVERS_INFOS[driver],
                         device_info=device_info)
```